### PR TITLE
[US267 Task296] Create chart from the data for found work

### PR DESCRIPTION
**Pull request by atharva-date**
found_work_data

### CODE REVIEW

File: react-ui/src/foundwork/index.js [hunk 1]
```diff
@@ -1,7 +1,53 @@
...
+        return acc;
+    }, {});
+    const chartData = Object.entries(tasksByDate).map(([date, count]) => ({
+        date,
+        count,
+    }));    
     return(
-        <div>
-            <p>{foundWorkData.length}</p>
+        <div align="center" width={750} height={300} sx={{ mt: 2 }}>
```
Comment (shahih25) [inline react-ui/src/foundwork/index.js]: You can keep it in **ResponsiveContainer** with width 100% and height 600.

Also, add a haeding at the top.

For reference: https://github.com/shahih25/SER516-Team-Clever/blob/main/react-ui/src/graph/linechart.js#L24

File: react-ui/src/foundwork/index.js [hunk 2]
```diff
@@ -1,7 +1,53 @@
...
+	Line,
+	XAxis,
+	YAxis,
+	Tooltip,
+	CartesianGrid,
+	Legend,
+} from 'recharts';
+
 const FoundWork = ({ foundWorkData }) => {
+    console.log(foundWorkData)
```
Comment (shahih25) [inline react-ui/src/foundwork/index.js]: nit: remove this log.

**Comment by atharva-date** [comment]
@shahih25 Changes implemented.. please check once

**Comment by shahih25** [comment]
Screenshot 2024-04-18 at 12 43 07 PM

**Review by shahih25 — APPROVED** [review]
lgtm

**Pull request merged**